### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by mattytrentini**
To include install instructions for virtualenv and refined the overview

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,29 @@
...
+guide them towards getting that first piece of code running that makes the
+hardware do something fun. From there it aims to provide the resources and
+guidance necessary to develop skills further and progress towards project-based
+learning.
 
 ## Installation
 
-To build the documentation locally, you will need [Sphinx](https://www.sphinx-doc.org). This is most easily retrieved via `pip install -r requirements.txt`. This should ideally be done in the virtual environment of your choice - although note that you may need to add generated folders to the `exclude_patterns` list in `conf.py` to avoid them being included and then complained about by Sphinx during build.
+[Sphinx](https://www.sphinx-doc.org) is used to build the documentation. To
+install using `virtualenv`:
```
Comment (HowManyOliversAreThere) [inline README.md]: This is actually using `venv`, a similar-but-different module (`virtualenv` is legacy)

File: README.md [hunk 2]
```diff
@@ -1,13 +1,29 @@
...
+
+```bash
+> python -m venv env
+> source env/bin/activate
+> pip install -r requirements.txt
```
Comment (HowManyOliversAreThere) [inline README.md]: Including the `>` in the code block means the command block isn't copy-pasteable - is the presenting in terminal style worth the slight hassle to the installer?

**Comment by HowManyOliversAreThere** [comment]
Couple minor comments but otherwise looks good

File: README.md (hunk 1 above)
Comment (mattytrentini) [reply README.md]: Good point, I'll update that.

File: README.md (hunk 2 above)
Comment (mattytrentini) [reply README.md]: I was 50/50, added the prompt at the last minute. I'll remove them!

**Comment by HowManyOliversAreThere** [comment]
All comments resolved, merging!

**Pull request merged**